plugin.video.mediathekview/resources/lib/mvutils.py

```diff
@@ -140,6 +143,26 @@ def file_remove(name):
             pass
     return False
 
+def deleteFiles(rootDir, regexp):
+    """
```

Comment:
Can you restrict this to the files being deleted in the method itself? Seems like an easy method to exploit/abuse

Reply:
it can just simply go away